base/pkg.yaml

```diff
@@ -4,6 +4,7 @@ shell: /toolchain/bin/bash
 dependencies:
   - image: docker.io/autonomy/tools:7de4db2
   - stage: musl
+    runtime: yes
```

Comment:
Cool, so this says "anytime `base` is a dependency, pull `musl` in too"?

Reply:
yep, exactly, and it works transitively and deduplicates stages as required.

we can actually pull this `tools` dependency as runtime and avoid putting it in every stage

I'm torn whether it's nice or not

Reply:
actually it's already pulled similar way by `finalize` which does copy `/` -> `/`, so `base` contains `tools`